product/views.py

```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
+from random import *
+import random
 
-# Create your views here.
+from django.http import JsonResponse
+from django.shortcuts import get_object_or_404
+from django.views import View
+
+from product.models import Product
+
+
+class DetailView(View):
```

Comment:
어떤 DetailView인지 명확하게 정의해주세요~
```suggestion
class ProductDetailView(View):
```

Reply:
확인 / 수정했습니다!